Swiftfin/Components/AlphaPicker/AlphaPickerCharacters.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2023 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import Foundation
+import SwiftUI
+
+enum AlphaPickerCharacters: String, CaseIterable, Displayable, Defaults.Serializable {
```

Comment:
This type is unnecessary. A better implementation would be to pass in the characters to the `AlphaPickerSelection`.

Reply:
Just so I understand. I should be calling UILocalizedIndexedCollation.current().sectionTitles.sorted() from the alphaPickerSelection in Default settings? I was originally looking at something like that but I didn't know if 1) that would work or if 2) calling a method there was allowed.

Reply:
We just don't need this intermediary type for a single static function that returns a `[String]`. The selector can take a `[String]` or it can be defined in the selector itself.